### DISCUSSION TITLE
Cannot virtualize Windows 11. qcow2 not found.

### DISCUSSION THREAD

**Discussion by neilroy1998** [utmapp/UTM#3966] [UTM Help (macOS)]
Downloaded and tried it in both UTM 3.1.5 and 3.1.4
Got the Windows 11 VHDX file (from the gallery section > microsoft site) and then clicked on '+' in UTM and virtualize according to the guide.
Gave the vhdx path and then added the new virtual machine.
But, now when I click on Start, it gives the following error:

**Error**

qemu-aarch64-softmmu: -drive if=none,media=disk,id=drive0,file=/Users/(username)/Library/Containers/com.utmapp.UTM/Data/Documents/Windows.utm/Images/Windows11_InsiderPreview_Client_ARM64_en-us_22598.qcow2,cache=writethrough: Could not open '/Users/(username)/Library/Containers/com.utmapp.UTM/Data/Documents/Windows.utm/Images/Windows11_InsiderPreview_Client_ARM64_en-us_22598.qcow2': No such file or directory

**Configuration**
* UTM Version: 3.1.4
* OS Version: MacOS 12.3.1
* Intel or Apple Silicon? Apple Silicon

**Comment by osy** [comment]
Can you check "/Users/(username)/Library/Containers/com.utmapp.UTM/Data/Documents/Windows.utm/Images/" and see if it has any files? When you created the VM did you wait a long time for the conversion to complete (it should have a spinning busy indicator for at least a minute"?

**Reply by neilroy1998** [reply]
That path only has a efi_vars.fd file in it.
When I created the VM, there was no wait time. I just added in the path to the vhdx file and clicked on Next, Next,...
Then it just finished. I donot think there was any qcow2 conversion.

**Reply by neilroy1998** [reply]
Uploading a gif for the issue...
[Win 11 install issue](https://imgur.com/a/70Z9yB3)

**Comment by mnryan11** [comment]
Hi neilroy1998, I wonder did you ever get a solution to your issue? I have exactly the same error now when I try install the Windows 11 22598 VDHX on my Mac. One thing I noticed was that the Windows11_InsiderPreview_Client_ARM64_en-us_22598.qcow2 was showing as not existing when I looked at the settings in UTM (using the option Open VM Settings on the summary screen) it was showing 0 size for this file. So I am wondering if the download of aRM64 from the Insider Preview worked correctly, however I did try to download twice and each time it showed with a six of 5.97GB on my Mac though various blogs suggest it should be 10GB plus. Did you have any luck with resolving this? Thanks Mike
